Symfony/CS/Fixer.php

```diff
@@ -117,6 +125,26 @@ public function getConfigs()
     }
 
     /**
+     * Get ErrorsManager instance.
```

Comment:
Also, I always prefer to write the human readable name, rather than the class names.